dask_glm/algorithms.py

```diff
@@ -134,12 +135,28 @@ def newton(X, y, max_iter=50, tol=1e-8, family=Logistic):
     return beta
 
 
-def admm(X, y, lamduh=0.1, rho=1, over_relax=1,
+def admm(X, y, reg=L1, lamduh=0.1, rho=1, over_relax=1,
```

Comment:
For ADMM, would this work with other regularizers? My understanding was that this algorithm is `L1` specific.  If this is for the sake of consistent API, it makes sense to keep it.